controller_manager/compile_commands.json

```diff
@@ -0,0 +1,682 @@
+[
```

Comment:
Why are we adding this file?

Reply:
My bad. This was autogenerated by qtcreator (I think).